src/components/cc-warning-payment/cc-warning-payment.js

```diff
@@ -97,25 +123,19 @@ export class CcWarningPayment extends LitElement {
       // language=CSS
       linkStyles,
       css`
+        
```

Comment:
is this space now necessary for stylelint ? (I don't have any opinion about this, i'm fine with or without it :smile:)